docs/src/projects/region-migration.md

```diff
@@ -12,16 +12,22 @@ aliases:
 Platform.sh doesn't offer an automated way to migrate a project from one region to another after it's created.
 However, the process to do so manually is fairly straightforward and can be scripted.
 
-## Why migrate between regions?
+## Why migrate between regions
 
 * Different data centers are located in different geographic areas,
   and you may want to keep your site physically close to the bulk of your user base for reduced latency.
```

Comment:
:bulb:
Do we want to also mention the environmental aspect of such a migration to newer regions? 

https://docs.platform.sh/development/regions.html#environmental-impact

Reply:
This is outside the scope of this task. Part of #2418

Reply:
Okey, should I add a note in there? Or is it part of the rework? (doesn't seem to be explicitly a part of it)

Reply:
It's part of the reasons for migration. It's already done.